test/Unit/Modules/Microsoft.Azure.Functions.PowerShellWorker.Tests.ps1

```diff
@@ -154,4 +154,60 @@ Describe 'Azure Functions PowerShell Langauge Worker Helper Module Tests' {
                 -Because 'The OutputBindings should be empty'
         }
     }
+
+    Context 'Trace-PipelineObject tests' {
+        BeforeAll {
+            $scriptToRun = @'
+    param($cmd, $modulePath)
+    Import-Module $modulePath
+    function Write-TestObject {
+        foreach ($i in 1..20) {
+            Write-Output $cmd
+        }
+        Write-Information '__LAST_INFO_MSG__'
+    }
+'@
+            $cmd = Get-Command Get-Command
+            $ps = [powershell]::Create()
+            $ps.AddScript($scriptToRun).AddParameter("cmd", $cmd).AddParameter("modulePath", $modulePath).Invoke()
+            $ps.Commands.Clear()
+            $ps.Streams.ClearStreams()
+
+            function Write-TestObject {
+                foreach ($i in 1..20) {
+                    Write-Output $cmd
+                }
+                Write-Information '__LAST_INFO_MSG__'
+            }
+        }
+
+        AfterAll {
+            $ps.Dispose()
+        }
+
+        AfterEach {
+            $ps.Commands.Clear()
+            $ps.Streams.ClearStreams()
+        }
+
+        It "'Trace-PipelineObject' Writes tracing to information stream while keeps input object in pipeline" {
```

Comment:
```suggestion
        It "Writes tracing to the information stream while keeping input objects in the pipeline" {
```

Since we already know what we're testing from the Context block. Rolls off the tongue a little better - "it writes tracing..."

Reply:
Fixed.